app/javascript/components/pxe-image-type-form/pxe-image-type-form.schema.js

```diff
@@ -0,0 +1,35 @@
+import { componentTypes, validatorTypes } from '@@ddf';
+import { parseCondition } from '@data-driven-forms/react-form-renderer';
+
+const createSchema = () => {
+  return ({
+    fields: [
+      {
+        component: componentTypes.TEXT_FIELD,
+        name: 'name',
+        id: 'name',
+        label: __('Name'),
+        maxLength: 255,
+      },
+      {
+        component: componentTypes.SELECT,
+        name: 'provision_type',
+        id: 'provision_type',
+        label: __('Type'),
+        placeholder: __('<Choose>'),
+        options: [
+            {
+              'label': 'Host',
+              'value': ':host'
```

Comment:
Should this be a symbol or a string?  Same below with `:vm`.  I thought the column returned a string, but I could be wrong.

Reply:
String, JS doesn't know how to deal with symbols...

Reply:
@DavidResende0 please drop the `:`